crontab/import_pg_catalog.php

```diff
@@ -67,7 +67,7 @@
 
 $trace = true;
 
-if ($trace) {
+if ($trace) { /** @phpstan-ignore-line */
```

Comment:
Let's move the commits with the special phpstan comments to the upcoming PR with the phpstan CI/CD integration, I'd rather not sprinkle the codebase with them until we have the checks in place to use them.

Reply:
OK. Will do.